python/lsst/ts/atmcssimulator/mcs_simulator.py

```diff
@@ -1219,7 +1217,8 @@ async def _write_telemetry(
         items = {k: v for k, v in data_dict.items() if not k.startswith("private")}
```

Comment:
Please change this to `data_dict = vars(data)`. I don't see how this can be a DDS topic (perhaps it is a holdover from when this was part of the CSC?). Please also delete the comment about Kafka.

Also I think you can skip the next line -- eliminating private fields. From looking at your code I think you only have public fields now for these "data" items (which I wanted a doc string so I knew what they were!).

Reply:
Thanks for catching this. It certainly makes the code simpler and better.